crates/binjs_meta/src/import.rs

```diff
@@ -69,13 +73,15 @@ impl Importer {
     ///     assert_eq!(contents_field.is_lazy(), true);
     /// }
     /// ```
-    pub fn import(s: &str) -> Result<SpecBuilder, weedle::Err<CompleteStr>> {
+    pub fn import<'a>(sources: &'a [&str]) -> Result<SpecBuilder, weedle::Err<CompleteStr<'a>>> {
```

Comment:
Perhaps better to use `sources: impl IntoIterator<Item = &str>` to support arbitrary iterables.

Reply:
Good point.